pkg/R/RDD.R

```diff
@@ -1100,9 +1137,7 @@ setMethod("saveAsObjectFile",
             if (!rdd@env$serialized) {
               rdd <- reserialize(rdd)
             }
-            .jcall(getJRDD(rdd), "V", "saveAsObjectFile", path)
-            # NOTE: rJava doesn't check for exceptions if the return type is void
-            .jcheck()
+            callJMethod(getJRDD(rdd), "saveAsObjectFile", path)
             # Return nothing
             invisible(NULL)
```

Comment:
While we're at it, we can make this `invisible` call take the last statement, as done in `setCheckpointDir`.